R/verb-pivot-longer.R

```diff
@@ -110,8 +110,21 @@ dbplyr_pivot_longer_spec <- function(data,
   spec <- deduplicate_spec(spec, data)
 
   id_cols <- syms(setdiff(colnames(data), spec$.name))
+  repair_info <- apply_name_repair_pivot_longer(id_cols, spec, names_repair)
+  id_cols <- repair_info$id_cols
+  spec <- repair_info$spec
 
   spec_split <- vctrs::vec_split(spec, spec[, -(1:2)])
+  nms_map <- tibble(
+    name = colnames(spec_split$key),
+    name_mapped = ifelse(
+      name %in% unlist(spec_split$key),
+      paste0("..", name),
```

Comment:
Why add `..` here?

Reply:
I just needed a temporary column name that does not yet exist, so I just prefixed the existing column name with `..`. Of course this column could already exist but this seemed unlikely enough to just do it like this

Reply:
Ok, makes sense.